tests/stdlib/topenssl.nim

```diff
@@ -0,0 +1,18 @@
+
+import unittest

```

Comment:
Please remove `unittest`

```nim
import openssl

# bug #16308
block:
  var rsa: PRSA
  var file: File

  if not file.open("topenssl_rsa_public_key.pem"):
    raise newException(Exception, "Error opening file")

  rsa = PEM_read_RSAPUBKEY(file, nil, nil, nil)
  if rsa == nil:
    raise newException(Exception, "Error reading public key")
  RSA_free(rsa)
  file.close()
```

Reply:
@xflywind  careful, try/finally or defer is needed otherwise you leak file when an exception (eg `raise newException(Exception, "Error reading public key")` ) is raised before file.close